Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Amazon.Lambda.TestTool.BlazorTester.csproj

```diff
@@ -6,7 +6,7 @@
     <OutputType>Exe</OutputType>
     <Description>A tool to help debug and test your .NET Core AWS Lambda functions locally.</Description>
     <LangVersion>Latest</LangVersion>
-    <VersionPrefix>0.13.1</VersionPrefix>
+    <VersionPrefix>0.13.2</VersionPrefix>
```

Comment:
Technically it is a feature so we should probably go up to `0.14.0`

Reply:
Done. Please review.